lib/still/compiler/incremental/node.ex

```diff
@@ -189,7 +201,6 @@ defmodule Still.Compiler.Incremental.Node do
       {:reply, error, state}
 
     :error, %PreprocessorError{} = error ->
-      Logger.error(inspect(error))
```

Comment:
What's the reason for this removal?

Reply:
Duplicate errors. We only need to error on the compile chain because all errors end up there.